src/shared/components/ResourceEditor/editorUtils.ts

```diff
@@ -76,22 +99,32 @@ export async function resolveLinkInEditor({
         resourceId: encodeURIComponent(url),
       });
       const entity = getOrgAndProjectFromResourceObject(data);
+      const isDownloadable = isDownloadableLink(data);
       return dispatchEvent(dispatch, {
         type: UISettingsActionTypes.UPDATE_JSON_EDITOR_POPOVER,
         payload: {
           ...defaultPaylaod,
           error: null,
           resolvedAs: 'resource',
           results: {
+            isDownloadable,
             _self: data._self,
             title: getResourceLabel(data),
             types: getNormalizedTypes(data['@type']),
-            resource: [
-              entity?.orgLabel,
-              entity?.projectLabel,
-              data['@id'],
-              data._rev,
-            ],
+            resource: isDownloadable
```

Comment:
I like the naming `isDownloadable`. It made this file change very easy to understand in the context of the PR :star: